CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to defund12.org
+
+Do you want to help with the project? Awesome! We can definitely use it. This project started off as the work of a single person but has exploded in several days to encompass a distributed team of volunteers all around the country (and maybe world?). We are still trying to figure out the best processes, it's a bit hectic!
+
+Check out the possible ways in which you could help below:
+
+## Writing emails (non-technical)
+
+If there is no email yet for your city, it would be awesome if you wrote one! See [the writing guide](docs/email_writing.md) for more info.
+
```

Comment:
We still a need for a triage team for manually closing and labeling incoming issues! If we follow this format, we can create a triage guide doc like `email_reviewing.md` and `email_formatting.md` with the process notes from `TEAMS.md`.